### PR TITLE
[Exchanges] add Bitget

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/715

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
ready to review !

File: Trading/Exchange/bitget/bitget_exchange.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    def is_supporting_exchange(cls, exchange_candidate_name) -> bool:
+        return cls.get_name() == exchange_candidate_name
+
+    async def create_order(self, order_type: trading_enums.TraderOrderType, symbol: str, quantity: decimal.Decimal,
+                           price: decimal.Decimal = None, stop_price: decimal.Decimal = None,
+                           side: trading_enums.TradeOrderSide = None, current_price: decimal.Decimal = None,
+                           params: dict = None) -> typing.Optional[dict]:
+        convert_quantity = order_type is trading_enums.TraderOrderType.BUY_MARKET
+        if convert_quantity:
+            # on Bitget, market orders are in quote currency (YYY in XYZ/YYY)
```
Comment (Herklos) [inline Trading/Exchange/bitget/bitget_exchange.py:54]: 😱

File: Trading/Exchange/bitget/bitget_exchange.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                                                                          **kwargs))
+
+    def _uniformize_trades(self, trades):
+        for trade in trades:
+            # fees example for paid fees in USDT:
+            # {'code': 'USDT', 'cost': -0.015922}
+            fee = trade[trading_enums.ExchangeConstantsOrderColumns.FEE.value]
+            if trading_enums.ExchangeConstantsFeesColumns.CURRENCY.value not in fee:
+                fee[trading_enums.ExchangeConstantsFeesColumns.CURRENCY.value] = fee.get("code")
+            fee[trading_enums.ExchangeConstantsFeesColumns.COST.value] *= -1
```
Comment (Herklos) [inline Trading/Exchange/bitget/bitget_exchange.py:84]: 😱

File: Trading/Exchange/bitget_websocket_feed/bitget_websocket.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.exchanges as exchanges
+import cryptofeed.defines as cryptofeed_constants
+from octobot_trading.enums import WebsocketFeeds as Feeds
+
+
+class BitgetCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
+    """
+    DISABLED FOR NOW
```
Comment (Herklos) [inline Trading/Exchange/bitget_websocket_feed/bitget_websocket.py:23]: 😱

**Review by Herklos — APPROVED** [review]
Nice work (and debug 😄 )!

**Review by Herklos — APPROVED** [review]

**Pull request merged**